.github/workflows/base_node_binaries.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           matrix_selection=$( jq -c '.[] | select( ."build_enabled" != false )' ${{ env.matrix-json-file }} )
           #
           # Setup the json build matrix
-          matrix=$(echo ${matrix_selection} | jq -s -c '{"builds": .}')
+          matrix=$(echo ${matrix_selection} | jq -s -c '{"networks": [{"type": "mainnet", "name": "mainnet"}, {"type": "testnet", "name": "esme"}], "builds": .}')
```

Comment:
Not sure if this is going to work as expected, would need to test. 

The idea of using json, was to keep vars that might be changed or different in an easy place, without changing the workflow direct. Might be an idea to rather do an include?